src/index.html

```diff
@@ -1,13 +1,20 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Antisnake</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Antisnake</h1>
-</body>
+  <html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Antisnake</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="container">
+      <div class="container__item container__item--red--1">1</div>
```

Comment:
```suggestion
      <div class="box box--1">1</div>
```
Try not to use such a long names for classes

Reply:
Добре, буду намагатися :)

До речі, в новому коміті я це виправив :)